client/src/plugins/error-tracking/ErrorTracking.js

```diff
@@ -112,6 +112,11 @@ export default class ErrorTracking extends PureComponent {
           return;
         }
 
+        // avoid sending element templates errors
+        if (isTemplateError(error)) {
```

Comment:
Where are these errors thrown? From the architecture perspective I'd rather make sure the errors do not bubble up as _errors_ but as _warnings_ (and thus do not get captured).

Reply:
The errors got thrown inside the properties panel Validator (cf. https://github.com/camunda/camunda-modeler/issues/2287#issuecomment-856764433). Handling them as warnings inside the Camunda Modeler might be a thing we should consider.

Reply:
I can barely remember @MaxTru was considering this as part of the last spring cleaning. Not sure what we come up with though.

Reply:
It just feels like a flaw to mix the element template concern into our core sentry error handling logic. An alternative could be to explicitly flag errors as warnings and ignore these (not send them to sentry). Either way we should not add template detection into that place :wink:

Reply:
We can probably add this concern to the BpmnEditor. I will find out, thanks for the recommendation 👍